cryptonite.cabal

```diff
@@ -231,6 +236,7 @@ Library
 
   Build-depends:     bytestring
                    , memory >= 0.14.14
+                   , random
```

Comment:
I couldn't find any reference to `random`, mistake ?

Reply:
I looked in the history of the file on master and it seems like that reference has not been there (at least) since I created my branch... If you think it is still needed, then I am fine with adding it, of course.

Reply:
Sorry if I wasnt clear, it just appear random is there as a new dependency, but doesnt seems used anywhere. Generally I dont want to have to depend on new things and random is no exception (specially its a particular bad random source and generator)

Reply:
Sorry, I interpreted your comment the wrong way. You're right: the reference to `random` is part of a leftover that I should have removed. It is fixed now.

Reply:
thanks !